Tests/scripts/slack_notifier.py

```diff
@@ -111,56 +114,137 @@ def options_handler():
     return options
 
 
-def get_failing_unit_tests_file_data():
-    failing_ut_list = None
+def get_artifact_data(artifact_relative_path: str) -> Optional[str]:
+    """
+    Retrieves artifact data according to the artifact relative path from 'ARTIFACTS_FOLDER' given.
+    Args:
+        artifact_relative_path (str): Artifact suffix.
+
+    Returns:
+        (Optional[str]): data of the artifact as str if exists, None otherwise.
+    """
+    artifact_data = None
     try:
-        file_name = f'{ARTIFACTS_FOLDER}/failed_lint_report.txt'
+        file_name = os.path.join(ARTIFACTS_FOLDER, artifact_relative_path)
         if os.path.isfile(file_name):
-            logging.info('Extracting lint_report')
-            with open(file_name, 'r') as failed_unittests_file:
-                failing_ut = failed_unittests_file.readlines()
-                failing_ut_list = [line.strip('\n') for line in failing_ut]
+            logging.info(f'Extracting {artifact_relative_path}')
+            with open(file_name, 'r') as file_data:
+                artifact_data = file_data.read()
         else:
-            logging.info('Did not find failed_lint_report.txt file')
+            logging.info(f'Did not find {artifact_relative_path} file')
     except Exception:
-        logging.exception('Error getting failed_lint_report.txt file')
-    return failing_ut_list
-
-
-def get_entities_fields(entity_title, report_file_name=''):
-    if 'lint' in report_file_name:  # lint case
-        failed_entities = get_failing_unit_tests_file_data()
-    else:
-        failed_entities = get_failed_steps_list()
-    entity_fields = []
-    if failed_entities:
-        entity_fields.append({
-            "title": f'{entity_title} - ({len(failed_entities)})',
-            "value": '\n'.join(failed_entities),
-            "short": False
-        })
-    return entity_fields
+        logging.exception(f'Error getting {artifact_relative_path} file')
```

Comment:
This was the behaviour before so i didnt want to change it. This is needed because the artifact will not exist if no test have failed if I remember correctly